lib/active_merchant/billing/network_tokenization_credit_card.rb

```diff
@@ -0,0 +1,16 @@
+module ActiveMerchant #:nodoc:
+  module Billing #:nodoc:
+    class NetworkTokenizationCreditCard < CreditCard
```

Comment:
Right, I'm seeing that now. :cool: (I'm cheating, we totally discussed this offline. ;))